src/components/attendee/Toggler/Toggler.module.scss

```diff
@@ -1,9 +1,14 @@
 @use "scss/attendee/space";
 @use "scss/attendee/theme";
+@use "scss/attendee/font";
+
+$-toggler-off-color: rgba(255, 255, 255, 0.3);
```

Comment:
If we put all these colors in `theme.scss`, we will have easier time later on determining what needs to be uniformed. This way it's harder to keep track and notice differences.

Reply:
done [a58b99b](https://github.com/sparkletown/sparkle/pull/2905/commits/a58b99bf83fb6ed1a9b39ec673fb2882dab9e8e9)